product_packaging_level/models/product_packaging_level.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019-2020 Camptocamp (<https://www.camptocamp.com>).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductPackagingLevel(models.Model):
+    _name = "product.packaging.level"
+    _description = "Level management for product.packaging"
+    _order = "sequence, code"
+
+    name = fields.Char(required=True, translate=True)
```

Comment:
Not blocking
I suppose a majority of projects using this module, will use one lang here.
If not, each of the concerns projects may make this fields translatable.
When you copy this record, copy translation comes with nightmares for consistency of naming
I know Odoo use this default settings. They demonstrate its capability. Not sure this is a good choice.

Reply:
You have no reason to copy a package level. You will configure a very limited amount of levels to use on all the product packagings.
As this can be very useful on e-commerc, it makes sense to have it translatable

Reply:
ok